testing/describes.js

```diff
@@ -441,6 +451,12 @@ class IntegrationFixture {
     /** @const */
     this.spec = spec;
 
+    this.spec.timeout = spec.timeout
+        || window.ampTestRuntimeConfig.mocha.integrationTimeout;
+    if (this.spec.retryOnSaucelabs === undefined) {
+      this.spec.retryOnSaucelabs = 3;
```

Comment:
Shall we make it 4 to match the `.retryOnSaucelabs()`

Reply:
SG. done